docs/getting-started/integrations-cdn/react-default-cdn.md

```diff
@@ -38,7 +38,228 @@ You get ready-to-use code tailored to your needs!
+	}
+
+	const {
+		ClassicEditor,
+		Bold,
+		Essentials,
+		Italic,
+		Mention,
+		Paragraph,
+		Undo,
+	} = cloud.CKEditor;
+
+	const { SlashCommand } = cloud.CKEditorPremiumFeatures!;
+
+	return (
+		<CKEditor
+			editor={ ClassicEditor }
+			data={ "<p>Hello world!!!</p>" }
+			config={ {
+				licenseKey: "<YOUR_LICENSE_KEY>",
+				toolbar: {
+					items: [ "undo", "redo", "|", "bold", "italic" ],
+				},
+				plugins: [
+					Bold,
+					Essentials,
+					Italic,
+					Mention,
+					Paragraph,
+					Undo,
+					SlashCommand,
+				],
+			} }
+		/>
+	);
+};
+```
+
+### Usage with CKBox
+
+To use `CKBox`, specify the version and theme (optionally) in the `useCKEditorCloud` configuration. Also, remember about the actual plugin configuration inside `<CKEditor/>` component.
+
+```js
+import React from "react";
+import { CKEditor, useCKEditorCloud } from "@ckeditor/ckeditor5-react";
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud( {
+		version: "{@var ckeditor5-version}",
+		ckbox: {
+			version: "2.5.1",
+			// Optional - it's already 'lark' by default.
+			theme: "lark",
+		},
+	} );
+
+	if ( cloud.status === "error" ) {
+		return <div>Error!</div>;
+	}
+
+	if ( cloud.status === "loading" ) {
+		return <div>Loading...</div>;
+	}
+
+	const {
+		ClassicEditor,
+		Bold,
+		Essentials,
+		Italic,
+		Mention,
+		Paragraph,
+		Undo,
+		CKBox,
+		CKBoxImageEdit,
+	} = cloud.CKEditor;
+
+	return (
+		<CKEditor
+			editor={ ClassicEditor }
+			data={ "<p>Hello world!!!</p>" }
+			config={ {
+				toolbar: {
+					items: [ "undo", "redo", "|", "bold", "italic" ],
+				},
+				plugins: [
+					Bold,
+					Essentials,
+					Italic,
+					Mention,
+					Paragraph,
+					Undo,
+					CKBox,
+					CKBoxImageEdit,
+				],
+				ckbox: {
+					tokenUrl: "https://api.ckbox.io/token/demo",
+					forceDemoLabel: true,
+					allowExternalImagesEditing: [ /^data:/, /^i.imgur.com\//, "origin" ],
+				},
+			} }
+		/>
+	);
+};
+```
+
+### Usage with external plugins
+
+There are various ways to use external plugins. Here is a list of them:
+
+* **Local UMD Plugins:** Dynamically import local UMD modules using the `import()` syntax.
+* **Local External Imports:** Load external plugins locally using additional bundler configurations (such as Vite).
+* **CDN 3rd Party Plugins:** Load JavaScript and CSS files from a CDN by specifying the URLs.
+* **Verbose Configuration:** Advanced plugin loading with options to specify both script and style sheet URLs, along with an optional `checkPluginLoaded` function to verify the plugin has been correctly loaded into the global scope.
+
+Here is an example:
+
+```js
+import React from 'react';
+import { CKEditor, useCKEditorCloud } from '@ckeditor/ckeditor5-react';
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud( {
+		version: '{@var ckeditor5-version}',
+		plugins: {
+			PluginUMD: async () => import( './your-local-import.umd.js' ),
+			PluginLocalImport: async () => import( './your-local-import' ),
```

Comment:
```suggestion
			PluginUMD: () => import( './your-local-import.umd.js' ),
			PluginLocalImport: () => import( './your-local-import' ),
```

Since there is no `await`, the `async` keyword should not be needed.

Reply:
You're right. But shouldn't this be the other way around - shouldn't we add `await`? This dynamic import is asynchronous AFAIR.

Reply:
tbh, I'd stay with `async` prefix without `await`. It's a little bit more verbose, but it's also more descriptive for the users which may be not super familiar with dynamic imports.

Reply:
Is it recommended? Won't such a syntax trigger some linter rules? @filipsobol

Reply:
It will trigger the [`require-await`](https://eslint.org/docs/latest/rules/require-await) rule if the end project has it enabled.

I suggested removing it only for technical correctness. I'm fine with keeping it if it makes things more clear. In this case, I don't think it's important for the end developer to know whether `import()` is async or not because it won't affect them in any way.